compiler/qsc/src/interpret/stateful/stepping_tests.rs

```diff
@@ -139,96 +138,76 @@ mod given_interpreter_with_sources {
         #[test]
```

Comment:
I can't comment on the line directly as it is outside the diff, but the module here is called `given_interpreter_with_sources` and that can probably be updated to something simpler like `given_sources` or `with_sources`